gcp_variant_transforms/libs/bigquery_schema_descriptor_test.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for infer_variant_header module."""
+
+
+from __future__ import absolute_import
+
+import unittest
+
+from apache_beam.io.gcp.internal.clients import bigquery
+from gcp_variant_transforms.libs import bigquery_schema_descriptor
+from gcp_variant_transforms.libs import bigquery_util
+
+
+class SchemaDescriptorTest(unittest.TestCase):
+  """Test case for :class:`SchemaDescriptor`."""
+
+  def setUp(self):
+    self._boolean = bigquery_util.TableFieldConstants.TYPE_BOOLEAN
+    self._float = bigquery_util.TableFieldConstants.TYPE_FLOAT
+    self._integer = bigquery_util.TableFieldConstants.TYPE_INTEGER
+    self._record = bigquery_util.TableFieldConstants.TYPE_RECORD
+    self._string = bigquery_util.TableFieldConstants.TYPE_STRING
+
+    self._nullable = bigquery_util.TableFieldConstants.MODE_NULLABLE
+    self._repeated = bigquery_util.TableFieldConstants.MODE_REPEATED
```

Comment:
nit: Another option instead of these assignments is to `import bigquery_util.TableFieldConstants as Consts` or something like that. I know the style guide does not approve individual class imports but we already do that in tests for exactly this reason of long names.

Reply:
Done!